src/amqp-ts.ts

```diff
@@ -561,11 +556,10 @@ export class Exchange {
         });
       }).catch((error) => {
         log.warn("Exchange publish error: " + error.message, { module: "amqp-ts", error });
-        var exchangeName = this._name;
         var connection = this._connection;
         return connection._rebuildAll(error).then(() => {
-          log.debug("Retransmitting message.", { module: "amqp-ts" });
-          return connection._exchanges[exchangeName].publish(content, routingKey, options);
+          log.warn("Connection rebuilt, NOT retransmitting message.", { module: "amqp-ts" });
```

Comment:
Do we still need this publish method here? When it is used? Because if it is not used, probably a good time to delete it.